### PR TITLE
Fix viewport query

### PR DESCRIPTION
**Pull request by pereirapleandro**
This PR was canceled...

### CODE REVIEW

**Comment by pedrolis** [comment]
Something that i missed when we find this issue was a `BadRequest` response when we used the parameters in wrong order.

What do you think guys about to add an extra validation when we create a value for `VIEWPORT` filter?

**Comment by pereirapleandro** [comment]
@pedrolis In this case, the Elasticsearch will return the details about the bad request. As an example, if you send a request with inverted parameters, you'll receive a response like this:
image

File: src/main/java/com/grupozap/search/api/adapter/FilterQueryAdapter.java [hunk 1]
```diff
@@ -272,8 +272,8 @@ public void apply(
               throw new UnsupportedFieldException(
                   fieldName, fieldType, FIELD_TYPE_GEOPOINT.toString(), VIEWPORT);
 
-            var topRight = new GeoPoint(filterValue.value(0, 1), filterValue.value(0, 0));
-            var bottomLeft = new GeoPoint(filterValue.value(1, 1), filterValue.value(1, 0));
+            var bottomLeft = new GeoPoint(filterValue.value(0, 1), filterValue.value(0, 0));
```
Comment (pedrolis) [inline src/main/java/com/grupozap/search/api/adapter/FilterQueryAdapter.java]: According to our wiki the filter viewport contract works this way:
``` 
field VIEWPORT [[topRightLon, topRightLat], [bottomLeftLon, bottomLeftLat]]
```

If I understood ok, you changed the contract to:
```
field VIEWPORT [[bottomLeftLon, bottomLeftLat], [topRightLon, topRightLat]]
```

If we're gonna change this, probably we need to check if there is currently any application using this, and modify it to the new contract. What do you think about it?

Comment (pereirapleandro) [reply src/main/java/com/grupozap/search/api/adapter/FilterQueryAdapter.java]: Yes! The contract was changed and we'll send these modifications for all clients through #announcements channel.

**Pull request closed without merging**